### ISSUE TITLE
Bump Go version to 1.12

### ISSUE THREAD

**Issue by izgeri** [cyberark/cloudfoundry-conjur-buildpack#24]
AC:
- [ ] Bump Go version for `conjur-env` to 1.12
- [ ] Convert from using `mitchellh/homedir` to `os.UserHomeDir`

**Comment by cadethacker** [comment]
I believe this was added to the v2.1.0 release, but also I believe it caused an issue on our side.  

During `cf push` this is what I'm seeing:

``` 
[cyberark-conjur-buildpack]: building the conjur-env binary.
   # github.com/cyberark/conjur-api-go/conjurapi
   /home/vcap/go/pkg/mod/github.com/cyberark/conjur-api-go@v0.6.0/conjurapi/config.go:122:15: undefined: os.UserHomeDir
   Failed to run all supply scripts: exit status 2
   Exit status 225
```

Here are the key stats of the `manifest.ym`

```
    buildpacks: 
    # v2.1.0 has the homedepot fixes in it
    - https://github.com/cyberark/cloudfoundry-conjur-buildpack.git#v2.1.0
    - https://github.com/cloudfoundry/java-buildpack.git#v4.19.1
```

The issue seems to be in an upstream module `conjur-api-go@v0.6.0` and not in the buildpack per se.

**Comment by cadethacker** [comment]
Also, it appears that you updated the golang version for the docker build for the offline buildpacks, but you also need to update the online build version of go (or make them look in the same spot for both)

https://github.com/cyberark/cloudfoundry-conjur-buildpack/blob/master/lib/install_go.sh#L8

I'm seeing this in the output:

```
 [cyberark-conjur-buildpack]: looking for conjur-env binary at /tmp/buildpackdownloads/f27cc09a6752e9b31da625ac90e9222f/vendor/conjur-env
    - need to compile conjur-env
   [cyberark-conjur-buildpack]: install go to build conjur-env binary
   -----> Download go 1.11.5
   /tmp/contents517841753/deps/0 ~
   [cyberark-conjur-buildpack]: download the conjur-env binary dependencies
```

**Comment by izgeri** [comment]
Thanks @cadethacker - @micahlee is looking into this right now

**Comment by micahlee** [comment]
Thanks @cadethacker, I've created a new issue for this (#39) and I'm putting in a PR to resolve it now.

**Comment by micahlee** [comment]
Thanks again @cadethacker. This has been fixed in [2.1.1](https://github.com/cyberark/cloudfoundry-conjur-buildpack/releases/tag/v2.1.1). Additionally, we've fixed our CI test bug that missed it in the first place.
